src/main/java/teammates/common/datatransfer/questions/FeedbackMsqQuestionDetails.java

```diff
@@ -403,9 +403,33 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
             return "";
         }
 
-        boolean isContainsNonEmptyResponse = false; // we will only show stats if there is at least one nonempty response
-
         Map<String, Integer> answerFrequency = new LinkedHashMap<String, Integer>();
+        int numChoicesSelected = getNumberOfEmptyResponse(responses, answerFrequency);
```

Comment:
Here you assign the return value of `getNumberOfEmptyResponse` to a variable called `numChoicesSelected`. Does it not seem odd to you? Either one of the names is inaccurate; could even be both.